api/src/main/java/org/openmrs/module/pihcore/task/PihCoreScheduledTaskExecutor.java

```diff
@@ -23,14 +21,15 @@ public class PihCoreScheduledTaskExecutor extends ScheduledExecutorFactoryBean {
     private final long tenMinutes = oneMinute * 10;
     private final long oneHour = oneMinute * 60;
     private final long fourHours = oneHour * 4;
+    private final long twentyfourHours = oneHour * 24;
 
     public PihCoreScheduledTaskExecutor() {
         setScheduledExecutorTasks(
                 task(fiveMinutes, fourHours, MarkAppointmentSchedulingAppointmentsAsMissedOrCompletedTask.class),
                 task(fiveMinutes, oneHour, PihCloseStalePullRequestsTask.class),
                 task(fiveMinutes, oneHour, PihCloseStaleCreateRequestsTask.class),
                 task(fiveMinutes, oneHour, PihCloseStaleVisitsTask.class),
-                task(fiveMinutes, oneHour, ClosePregnancyProgramTask.class),
+                task(fiveMinutes, twentyfourHours, ClosePregnancyProgramTask.class),
```

Comment:
It occurred to me that there is no need (besides to ease testing :) to run this task more than once per day.

Reply:
I was going to make the same comment, but then I figured we must do this for a reason, as the same is true for most/all of the other tasks.  In particular, if the system is not reliably up 24/7, I'd worry that the task might not execute every day if the server was down at the given execution time each day.

Reply:
The other tasks are more time sensitive, as they are all on the order of hours (close a visit after 12 hours with no encounters vs close a program after 6 weeks in a certain state).  And all these tasks are set to run for the first time 5 minutes after startup, so we shouldn't need to worry about the something not running because the server is down?  @mseaton   Easy enough to move it to 4 or 12 hours if that feels safer, though.

Reply:
I think I'll change this to twelve hours...